keyboards/genone/eclipse_65/keymaps/via/keymap.c

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 GEN ONE LLC 
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {

```

Comment:
VIA requires that all of the layers be populated.  

You need to add 2 more layers (for a total of 4), or set the max layers to 2

Reply:
Updated the keymap.c file to include 4 layers as requested.